### PR TITLE
Fix mobile number not required bug

### PR DESCRIPTION
**Pull request by nathanbegbie**
See https://praekeltorg.atlassian.net/browse/MOLO-378

### CODE REVIEW

**Comment by nathanbegbie** [comment]
@codiebeulaine or @Ntu2koTrevor please review

**Comment by g1039** [comment]
👍

**Comment by g1039** [comment]
👍

File: molo/profiles/forms.py [hunk 1]
```diff
@@ -162,10 +162,11 @@ def is_valid(self):
                 settings = SettingsProxy(site)
                 profile_settings = settings['profiles']['UserProfilesSettings']
                 number = self.data['mobile_number']
-                if number.startswith('0'):
-                    number = number[1:]
-                number = profile_settings.country_code + \
-                    number
+                if number != '':
```
Comment (miltontony) [inline molo/profiles/forms.py:165]: @nathanbegbie a better check will be `if number` which will ensure that you catch the `None` case too

**Pull request merged**